includes/services/GoogleMaps3/Maps_GoogleMaps3.php

```diff
@@ -318,6 +318,7 @@ protected function getDependencies() {
 	public static function getApiScript( $langCode, array $urlArgs = [] ) {
 		$urlArgs = array_merge(
 			[
+				'key' => $GLOBALS['egMapsGMaps3ApiKey'],
```

Comment:
What will happen for existing users for which the maps work, but that do not have a key set? Won't this add `&key=`, which might then cause the map to fail?